lib/pages/wp-home-page.js

```diff
@@ -12,6 +12,10 @@ export default class WPHomePage extends AsyncBaseContainer {
 		super( driver, by.css( 'body' ), url );
 	}
 
+	async _postInit() {
```

Comment:
This is good since it runs against wordpress.com, but not _every_ sign up test uses this page (only paid sign ups) therefore not every sign up test will have the banner dismissed